mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -44,3 +44,4 @@ data:
   check-labels.label-file: "/gitrepos/kubernetes/labels.yaml"
   issue-categorizer.triager-url: "http://issue-triager-service:5000"
   cla.cla-status-context: ""
+  alias.alias-file: "/gitrepos/kubernetes/OWNERS_ALIASES"
```

Comment:
We should probably change [mungegithub/features/aliases.go#L96](https://github.com/kubernetes/contrib/blob/b70447aa59ea14468f4cd349760e45b6a0a9b15d/mungegithub/features/aliases.go#L96).